libraries/Microsoft.Bot.Schema/Teams/BotMeetingNotificationChannelData.cs

```diff
@@ -7,25 +7,25 @@ namespace Microsoft.Bot.Schema.Teams
     using Newtonsoft.Json;
 
     /// <summary>
-    /// Container for list of <see cref="OnBehalfOf"/> information for a meeting <see cref="TeamsMeetingNotificationChannelData"/>.
+    /// Specify Teams Bot meeting notification channel data.
     /// </summary>
-    public partial class TeamsMeetingNotificationChannelData
+    public partial class BotMeetingNotificationChannelData
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="TeamsMeetingNotificationChannelData"/> class.
+        /// Initializes a new instance of the <see cref="BotMeetingNotificationChannelData"/> class.
         /// </summary>
-        public TeamsMeetingNotificationChannelData()
+        public BotMeetingNotificationChannelData()
         {
             CustomInit();
         }
 
         /// <summary>
-        /// Gets or sets the <see cref="OnBehalfOf"/> for user attribution.
+        /// Gets or sets the list of <see cref="OnBehalfOf"/> for user attribution.
         /// </summary>
-        /// <value>The meeting notification's <see cref="OnBehalfOf"/>.</value>
+        /// <value>The Teams Bot meeting notification's <see cref="OnBehalfOf"/>.</value>
 #pragma warning disable CA2227 // Collection properties should be read only (we can't change this without breaking binary compat)>
         [JsonProperty(PropertyName = "OnBehalfOf")]
-        public IList<TeamsMeetingNotificationOnBehalfOf> OnBehalfOf { get; set; }
+        public IList<OnBehalfOf> OnBehalfOf { get; set; }
```

Comment:
Hmm this is an indeed interesting problem. As this is a list of OnBehalfOf entity, I think we should update the property name to reflect this is a list. (ex: On SMBA, it seems like we say OnBehalfOfEntities). 

Then how a developer will use is.. 
They will create BotMeetingNotificationChannelData.OnBehalfOfentities = new List<OnBehalfOf>{
    new OnBehalfOf {...., ... }

Reply:
Let's discuss this in the Teams chat.

Reply:
Make sense. I can update it to `public IList<OnBehalfOf> OnBehalfOfList { get; set; }`
@YunnyChung , what do you think about the property name?

Reply:
According to the discussion offline, Yunny agreed to update the proposal. The update is in the next iteration.